docs/en/start-v2/locally/deployment.md

```diff
@@ -34,13 +34,13 @@ tar -xzvf "apache-seatunnel-incubating-${version}-bin.tar.gz"
 Since 2.2.0-beta, the binary package does not provide connector dependencies by default, so when using it for the first time, you need to execute the following command to install the connector: (Of course, you can also manually download the connector from [Apache Maven Repository](https://repo.maven.apache.org/maven2/org/apache/seatunnel/) to download, then manually move to the `connectors/seatunnel` directory).
 
 ```bash
-sh bin/install-plugin.sh 2.3.2
+bash bin/install-plugin.sh 2.3.2
```

Comment:
@javalover123 

`sh` has better compatibility, it seems the `bad substitution` error should be fixed